src/main/java/org/sonarsource/scanner/maven/bootstrap/MavenUtils.java

```diff
@@ -113,4 +116,38 @@ private static String escapeCommas(String value) {
     return value.contains(",") ? ("\"" + value + "\"") : value;
   }
 
+  public static List<String> splitAsCsv(String joined) {
```

Comment:
We use StringUtils elsewhere in the code. Can we replace the custom implem with a method call?